source/systems/common.rst

```diff
@@ -64,5 +64,1789 @@ RDHPCS Platforms
+and how accurately idle jobs and running jobs have specified their wall clock
+time.
+
+Deleting jobs
+-------------
+
+To cancel a job use the scancel command
+
+.. code-block:: shell
+
+   $ scancel $JOBID
+
+Job Preemption
+--------------
+
+During Hurricane season (usually from July 1 thru Oct 31) job preemption may be
+in effect for some reservations. All users will be notified when preemption is
+in effect and on what partition(s) it exists. Preemption means that non
+realtime jobs are eligible to be preempted (killed) to allow the project with
+preemption rights on a certain set of nodes to run at a set time. Preemption
+only occurs when the scheduler is unable to run a realtime job due to lack of
+space. Jobs with less wall clock time remaining are less likely to be
+preempted. Users should, therefore, avoid requesting more wall clock time than
+is necessary in order to minimize their chances of being preempted. Realtime
+jobs running under a standing reservation or in a realtime queue are not
+eligible for preemption as they already have their and nodes and time slot
+reserved. When jobs are preempted, they are killed and automatically requeued
+at the same priority they had when they previously started to run. If you do
+not wish your job to be requeued when it is preempted, you must submit it using
+the &quot;--no-requeue&quot; option.
+
+Useful variables in the batch environment
+-----------------------------------------
+
+The variables listed below are created in your job's environment by the batch
+system. Developers have found these useful as ways to make batch scripts more
+intelligent and limit hard-coding information about your job in the
+environment.
+
+.. code-block:: shell
+
+   $SLURM_JOB_ID - The jobid of the currently running job
+   $SLURM_SUBMIT_DIR - The directory from which the batch script was submitted
+   $SLURM_JOB_QOS - The assigned quality of service (qos) for this job
+   $SLURM_NTASKS - The number of tasks assigned to this job
+
+Charging Algorithm for Your Jobs
+================================
+
+Most jobs are charged only based on the amount of wall clock time **used** by
+the jobs, not based on the amount of time **requested** for the jobs. Exception
+to this is a rule are jobs run in a reservation, which are charged based on the
+length of the reservation.
+
+The units allocated and the numbers reported by the command
+<code>saccount_params</code> are in **core-hours**.
+
+Charging is done differently for serial and parallel jobs and serial jobs.
+
+In the notes below, we will use **CPN** as "Cores Per Node"; this number
+depends on the machine and the partition you are running on, and here are some
+examples based on the current hardware:
+
+* Hera - 40 cores per node
+* xJet - 24 cores per node
+* sJet - 16 cores per node, etc
+
+**Charging for parallel jobs** (any job requesting two or more cores):
+
+**Core-hours = (actual wall clock time used by the job) * (number of nodes
```

Comment:
Use the `.. math::` directive:

```rst
.. math:: Core\_Hours = WallClock\_used * Nodes * Cores\_per\_Node
```

Reply:
Thanks for introducing me to the math directive. I added as you wrote it -- the equation is centered and italicised.